package/yast2-installation-control.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Oct 17 09:53:22 UTC 2017 - snwint@suse.com
+
+- adjust control.rnc for storage-ng
```

Comment:
JFYI: the package autosubmission will fail without a bugzilla/fate number in changes.